config/deb.am

```diff
@@ -38,9 +43,14 @@ if CONFIG_USER
 	pkg7=$${name}-test-$${version}.$${arch}.rpm; \
 	pkg8=$${name}-dracut-$${version}.$${arch}.rpm; \
 	pkg9=$${name}-initramfs-$${version}.$${arch}.rpm; \
+	path_prepend=`mktemp -d /tmp/intercept.XXX`; \
+	ln -s /bin/true $${path_prepend}/dh_shlibdeps; \
```

Comment:
> This method is fundamentally broken

Maybe, but let's not make it more so

> if you simply uninstall all zfsonlinux packages, dh_shlibdeps will not recognize the libraries (since they are part of no installed package), and will, in fact, generate no dependencies

This is not entirely correct: you can see from my first comment that libuutil1 depends on `libuuid1 (>= 2.16)`. With this change we're blowing away this bit of information. What happens when you try to downgrade libuuid? Does the package manager warns you that zfs needs it or just drives on and replace the library with some other version that may, or may not, be compatible? I wouldn't trust a system running with these packages, it just seems an accident waiting to happen, we're better off `make install`ing zfs direclty instead.

> I most definitely agree this approach is far from perfect

Then let's not make this a default for everyone until it's a better choice than the current one: can't we esclude just the conflicting libraries?